packages/nextjs/components/HooksExample.tsx

```diff
@@ -0,0 +1,70 @@
+import { ReactNode } from "react";
+
+const Red = ({ children }: { children: ReactNode }) => {
+  return <span className="text-[#FF5962]">{children}</span>;
+};
+const Cyan = ({ children }: { children: ReactNode }) => {
+  return <span className="text-[#00B0A6]">{children}</span>;
+};
+
+const Purple = ({ children }: { children: ReactNode }) => {
+  return <span className="text-[#7B61FF]">{children}</span>;
+};
+export const HooksExample = () => {
+  return (
+    <div className="w-full flex-grow  bg-white rounded-2xl p-6 drop-shadow-[0px_0px_10px_rgba(139,178,241,0.50)]">
+      <div className="bg-base-300/40 w-full h-full border border-primary rounded-3xl p-4 lg:p-6 text-xs lg:text-sm  font-mono overflow-x-scroll whitespace-nowrap  lg:overflow-auto lg:whitespace-normal">
+        <p className="my-3">
+          <Red>import</Red>
+          {" { "} useScaffoldContractWrite {" } "}
+          <Red>from</Red> <Cyan>&quot; ~~/hooks/scaffold-eth &quot;</Cyan>;
+        </p>
+
+        <p className="my-3">
+          <Red>const</Red>
+          {" { "}writeAsync, isLoading, isMining{" } "}
+          <Cyan> = </Cyan>
+          <br />
+          <Purple>useScaffoldContractWrite</Purple>({"{"}
+          <br />
+          <span className="inline-block ml-4">
+            <Cyan>contractName</Cyan>: <Cyan> &quot;YourContract&quot;</Cyan>,
+            <br />
+            <Cyan>functionName</Cyan>: <Cyan> &quot;setPurpose&quot;</Cyan>,
+            <br />
+            <Cyan>args</Cyan>: [<Cyan>&quot;The value to set&quot;</Cyan>],
+            <br />
+            <Cyan>blockConfirmation</Cyan>: <Cyan>1</Cyan>,
+            <br />
+            <Purple>onBlockConfirmation</Purple>: (txnReceipt) =&gt; {"{"}
+            <br />
+            <span className="inline-block ml-4">
+              console.<Purple>log</Purple>(<Cyan>&quot;Transaction blockHash&quot;</Cyan>, &nbsp;txnReceipt.
+              <Cyan>blockHash</Cyan>
```

Comment:
Same for this one, I expected `blockHash` to have the same color as `txnReceipt`, but maybe is just what I'm used to.